pallets/phala/src/compute/wrapped_balances.rs

```diff
@@ -220,18 +486,21 @@ pub mod pallet {
 		#[frame_support::transactional]
 		pub fn unwrap_all(origin: OriginFor<T>) -> DispatchResult {
 			let user = ensure_signed(origin)?;
+			ensure!(
+				!SlashDebts::<T>::contains_key(&user),
```

Comment:
I suggest to check `SlashDebts::<T>::get(&user) > Zero::zero()`. So you won't need to worry about if a zero debt should be deleted.

Reply:
done